src/Microsoft.ML.StandardLearners/Standard/MultiClass/MetaMulticlassTrainer.cs

```diff
@@ -121,11 +118,7 @@ protected IDataView MapLabelsCore<T>(ColumnType type, InPredicate<T> equalsTarge
 
         protected TScalarTrainer GetTrainer()
         {
-            // We may have instantiated the first trainer to use already, from the constructor.
-            // If so capture it and set the retained trainer to null; otherwise create a new one.
-            var train = _trainer ?? CreateTrainer();
-            _trainer = null;
-            return train;
+            return _trainer ?? CreateTrainer();
```

Comment:
>CreateTrainer [](start = 31, length = 13)

Can we assign this to _trainer in the constructor? This way we create the trainer only once, and we won't need the GetTrainer() method.